library/Zend/Paginator/PaginatorIterator.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Zend\Paginator;
+
+use OuterIterator;
+
+class PaginatorIterator implements OuterIterator
```

Comment:
To bo honest I do not see why.
I noticed that all props and methods have their own docs, even if it's just a copy-paste from php.net.
If docs for class should be done in separate repo then, imo, it should be done by author of PR.